js-runtime/index.js

```diff
@@ -1065,14 +1065,14 @@ module.exports = {
     const nskey = wth[0];
     const a = wth[1];
     const b = ds[`${nskey[0]}:${nskey[1]}`];
-    return [true, await func(a, b)];
```

Comment:
I hurt myself in my confusion when I swapped the argument order like @cdmistman suggested (which makes a lot of sense! but my js was already written the other way around).